src/snyk/snykCode/error/snykCodeErrorHandler.ts

```diff
@@ -132,6 +143,11 @@ export class SnykCodeErrorHandler extends ErrorHandler implements ISnykCodeError
       msg += `. ${options}`;
     }
 
+    // logging options to sentry as well
+    if (error instanceof Error && error) {
+      error.message = msg;
```

Comment:
Out of curiosity - how does `error.message` look here? An example would be nice to understand what Sentry will report 😄

Reply:
Didn't really like this pattern tbf. 

Going with pattern defined here https://blog.sentry.io/2019/01/31/using-nginx-sentry-trace-errors-logs

Reply:
Didn't really like this pattern tbf. 

Going with pattern defined here https://blog.sentry.io/2019/01/31/using-nginx-sentry-trace-errors-logs